src/pages/_app.tsx

```diff
@@ -93,14 +93,6 @@ const MyApp: FC<AppProps> = ({Component, pageProps}) => {
         <title>STROM</title>
         <meta name="viewport" content="width=device-width, initial-scale=1" />
       </Head>
-      {/* ak pouzijeme vsade <Typography>, nemuseli by sme mat takyto globalny styl... ale je to teda
-          safe a krajsie, nech kazdy maly (aj debug) textik je aspon nasim fontom a ne v Times :D
-          source: https://nextjs.org/docs/pages/building-your-application/optimizing/fonts#apply-the-font-in-head */}
-      <style jsx global>{`
-        html {
-          font-family: ${theme.typography.fontFamily};
-        }
-      `}</style>
```

Comment:
myslis ze uz vsade mame naozaj pouzite Typography? :-D

Reply:
nope. ale testoval som, ze CssBaseline pridava font-family na body, aj ked to v tom MUI kode nevidim... 😄 takze sa toho mozeme safely zbavit